ui/desktop/src/components/ProviderGuard.tsx

```diff
@@ -280,103 +209,64 @@ export default function ProviderGuard({ didSelectProvider, children }: ProviderG
                 </div>
                 <p className="text-text-muted text-base sm:text-lg mt-4 sm:mt-6">
                   Since it's your first time here, let's get you setup with a provider so we can
-                  make incredible work together. Scroll down to see options.
+                  make incredible work together.
```

Comment:
can we fix this text too?

Reply:
drop the scroll down, this should all fix the current window. setup is two words. I don't know the user knows what a provider is. so maybe:

Since it’s your first time here, let’s get you set up with an AI provider so goose can work its magic.